rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -26,6 +26,9 @@
 public class FederationSupportImpl implements FederationSupport {
 
     private static final Logger logger = LoggerFactory.getLogger(FederationSupportImpl.class);
+    private static final int FEDERATION_NON_EXISTENT_CODE = FederationChangeResponseCode.FEDERATION_NON_EXISTENT.getCode();
+    private static final int FEDERATION_CHANGE_SUCCESSFUL_CODE = FederationChangeResponseCode.SUCCESSFUL.getCode();
+    private static final int GENERIC_ERROR_CODE = FederationChangeResponseCode.GENERIC_ERROR.getCode();
```

Comment:
Not a big fan of copying enum values into local variables. Why not use the enum directly? You can import it directly if you want to make the lines shorter